Modules/_csv.c

```diff
@@ -1213,6 +1213,19 @@ csv_writerow(WriterObj *self, PyObject *seq)
             append_ok = join_append(self, field, quoted);
             Py_DECREF(field);
         }
+        else if (PyBool_Check(field)) {
+            PyObject *str;
+            double d = PyFloat_AsDouble(field);
+            str = PyFloat_FromDouble(d);
```

Comment:
@serhiy-storchaka 
Thank you for your kind review.
So in this case, This code should be go this way?

````
            PyObject *str;
            double d = PyFloat_AsDouble(field);
            str = PyFloat_FromDouble(d);
            if (str == NULL) {
                Py_DECREF(iter);
                return NULL;
            }
            str = PyObject_Str(str);
            Py_DECREF(str);
            Py_DECREF(field);
            if (str == NULL) {
                Py_DECREF(iter);
                return NULL;
            }
            append_ok = join_append(self, str, quoted);
            Py_DECREF(str);
````